prov/verbs/src/verbs_msg.c

```diff
@@ -212,6 +345,19 @@ struct fi_ops_msg fi_ibv_msg_ep_msg_ops = {
 	.injectdata = fi_ibv_msg_ep_injectdata,
 };
 
+struct fi_ops_msg fi_ibv_msg_ep_sstream_ops = {
+	.size = sizeof(struct fi_ops_msg),
+	.recv = fi_ibv_sstream_ep_recv,
+	.recvv = fi_ibv_sstream_ep_recvv,
+	.recvmsg = fi_ibv_sstream_ep_recvmsg,
+	.send = fi_ibv_sstream_ep_send,
+	.sendv = fi_ibv_sstream_ep_sendv,
+	.sendmsg = fi_ibv_sstream_ep_sendmsg,
+	.inject = fi_ibv_msg_ep_inject,
+	.senddata = fi_no_msg_senddata,
+	.injectdata = fi_no_msg_injectdata,
```

Comment:
injectdata and senddata are not supported by stream eps, since there's no peer CQ to write the data to.

Reply:
This patch reflects that right?